modular_skyrat/master_files/code/datums/id_trim/syndicate.dm

```diff
@@ -18,8 +18,8 @@
 	assignment = "DS-2 Operative"
 	trim_state = "trim_syndicateoperative"
 
-/datum/id_trim/syndicom/skyrat/assault/assistant
-	assignment = "Operative"
+/datum/id_trim/syndicom/skyrat/assault/miner //assistants when they are forced to work:
+	assignment = "Mining Officer"
```

Comment:
Heads up, it might be an idea to append DS2 to the DS2 roles so they aren't confused.

Reply:
I believe none of the DS2 jobs actually have a DS2 suffix. Besides, they're on the black ID cards, figures it's easy to figure out IC.
Unless you meant codewise, like '/datum/id_trim/syndicom/skyrat/assault/ds2miner' ?

Reply:
Going to turn the 'skyrat/assault' part into 'skyrat/ds2'